src/main/java/org/jabref/Globals.java

```diff
@@ -56,13 +56,11 @@
     /**
      * Manager for the state of the GUI.
      */
-
-    public static ClipBoardManager clipboardManager = new ClipBoardManager();
-
     public static StateManager stateManager = new StateManager();
     public static ExporterFactory exportFactory;
     public static CountingUndoManager undoManager = new CountingUndoManager();
     public static BibEntryTypesManager entryTypesManager = new BibEntryTypesManager();
+    public static ClipBoardManager clipboardManager = ClipBoardManager.INSTANCE;
```

Comment:
That's a bit of a controversial topic (also among JabRef's devs) but we try to avoid to use the `Singleton` pattern now. It makes it harder to test code as you cannot mock or replace the instance (as it's a final field and the constructor private). Moreover, the code should always use the manager defined here in `Globals` instead of `Clipboardmanager.instance` and there is no Java language feature that helps us here. 

So in summary I would like to ask you to revert this change.

Reply:
I will ll do, no prob. I supposed that it was a risky decision... :sweat_smile:

Reply:
It's really good that you went for it! Learning by doing is the best approach for these software architecture questions.

Reply:
Good good. I got a working snapshot without using the singleton pattern. I think the `addX11Support()` method should be static, so I also switched to static attributes (this way, we can access from static and non-static methods). I await your feedback!